modules/fbx/fbx_document.cpp

```diff
@@ -2048,6 +2048,7 @@ Error FBXDocument::_parse(Ref<FBXState> p_state, String p_path, Ref<FileAccess>
 		opts.ignore_embedded = true;
 	}
 	opts.generate_missing_normals = true;
+	opts.use_blender_pbr_material = true;
```

Comment:
This option compensates for a change in defaults in ufbx:

By default ufbx always tried to detect if a file was exported from Blender, and if so would parse the FBX Phong `ReflectionFactor` property and texture as "metalness", which is definitely not what most other importers do, but matches how Blender exports materials. I disabled this by default as this behavior may be confusing.

Any thoughts whether we should enable this or not?

Reply:
My opinion is anything that helps the Blender -> Godot Engine workflow is welcome, but what are the implications of this? What could cause trouble?

Reply:
It doesn't have too many downsides, I think the major two are:

* This behavior is different than any other importer that I know (except possibly Blender's own one, have not checked in detail)
* If a Blender file is imported and re-exported into other software (eg. FBX converter or something), ufbx won't know the file is originally exported from Blender and this hack won't activate, meaning the file loses the metalness textures.

Reply:
> If a Blender file is imported and re-exported into other software (eg. FBX converter or something), ufbx won't know the file is originally exported from Blender and this hack won't activate, meaning the file loses the metalness textures.

This feels like a deal breaker..

Reply:
Yeah that's one of the reason why I removed it as default. It's the tradeoff between convenient, but in rare cases surprising behavior, and less convenient but more stable (and standard) behavior.